src/components/button/button.spec.js

```diff
@@ -412,4 +412,42 @@ describe('Button', () => {
       }, buttonWithServiceIcon.toJSON(), { modifier: `${StyledIcon}` });
     });
   });
+
+  describe('when the fullWidth prop is provided', () => {
+    it('does not throw when size is large and buttonType is `secondary`', () => {
```

Comment:
This will never throw now the `invariant` has been taken out and any `size` and `buttonType` can be `fullWidth`

Reply:
Good spot. I thought I removed the invariant tests but I guess I missed this. Thank you for spotting it.